src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerConfigurations/AnonymizerConfiguration.cs

```diff
@@ -19,6 +19,12 @@ public class AnonymizerConfiguration
         // Static default crypto hash key to provide a same default key for all engine instances
         private static readonly Lazy<string> s_defaultCryptoKey = new Lazy<string>(() => Guid.NewGuid().ToString("N"));
 
+        private static readonly string s_defaultPresidioAnalyzerUrl = "http://127.0.0.1:5002";
+
+        private static readonly string s_defaultPresidioAnonymizerUrl = "http://127.0.0.1:5001";
+
+        private static readonly string s_defaultPresidioAnalyzedLanguage = "en";
```

Comment:
NIT: Analyzer instead of Analyzed?

Reply:
@SharonHart

Reply:
I didn't name it, your call..

Reply:
yep, change to Analyzer to be consistent with the rest of the parameters names